particles/Tutorial/5_Collections/collections.js

```diff
@@ -0,0 +1,26 @@
+defineParticle(({DomParticle, html}) => {   
+  return class extends DomParticle {
+    get template() {
+      // TODO: Update this tutorial to show how to output a list of things in a template.
```

Comment:
Have a look at src/runtime/test/artifacts/Common/source/List.js for a template example. particles/Profile/source/FavoriteFoodPicker.js has a more complex example.

Reply:
Thanks for that! I've updated the tutorial with that, PTAL